src/components/AudioSystemBanner/AudioSystemBanner.scss

```
@@ -6,7 +8,10 @@
   padding: 6px 12px;
   font-size: 12px;
   color: #fff;
-  background-color: #e67e22;
+  // NOTE: this is a degraded state wearing the usage amber. Left on the same
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)